packages/wonder-blocks-form/src/components/__tests__/labeled-text-field.test.tsx

```diff
@@ -729,10 +729,10 @@ describe("Required LabeledTextField", () => {
             "test-labeled-text-field-field",
         );
         textField.focus();
-        await userEvent.clear(textField);
```

Comment:
Since we now clear the error state when the field is cleared, this test case can be updated with an example where we update the input value instead of clear it